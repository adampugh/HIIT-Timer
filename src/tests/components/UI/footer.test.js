import React from "react";
import { shallow } from "enzyme"; 

import Footer from "../../../components/UI/footer";

test("should render footer correctly", () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper).toMatchSnapshot();
});