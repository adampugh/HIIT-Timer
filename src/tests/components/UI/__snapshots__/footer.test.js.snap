// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render footer correctly 1`] = `
<footer>
  <div
    className="container"
  >
    <h1>
      HIIT Timer
    </h1>
    <socialLinks />
    <h3>
      Â© 2018 Design by 
      <a
        href="https://adampugh.github.io/kokonoka/"
        rel="noopener noreferrer"
        target="_blank"
      >
        Kokonoka 
      </a>
    </h3>
  </div>
</footer>
`;
